<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactInfo</class>
 <widget class="QMainWindow" name="ContactInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_contactus">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contact Us</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_github">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Github</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_discord">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Discord</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_URL">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Website</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Send us an Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_github">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Check out the Repository on GitHub</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_discord">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Visit our Support Server on Discord</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_website">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Visit our Website</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
